src/components/common/form/message-summary/MessageSummary.spec.jsx

```diff
@@ -6,24 +6,23 @@ import MessageSummary from './MessageSummary';
 describe('MessageSummary', () => {
   window.HTMLElement.prototype.scrollIntoView = jest.fn();
   it('should render a summary component with all messages details', () => {
-    const testMessages = {
-      delete: { inputName: 'delete', message: 'Row Deleted' },
-      update: { inputName: 'update', message: 'Row Updated' },
-      insert: { inputName: 'insert', message: 'Row Inserted' },
+    const summaryMessages = {
```

Comment:
I know we don't have multiple messages right now, but should there be a test for the scenario for multiple summary messages being displayed at once to test the map function?

Reply:
I've added a test to show we can handle multiple messages being input, but since we can only have one of each key and only handle one key, it won't show the second message